### DISCUSSION TITLE
I would like to add a photoshop white ink spot color channel to cmyk's tif file. Is there a way to do this

### DISCUSSION THREAD

**Discussion by 291598912** [ImageMagick/ImageMagick#6304] [Digital Image Processing]
I would like to add a photoshop white ink spot color channel to cmyk's tif file. Is there a way to do this

**Comment by snibgo** [comment]
IM can't add spot colours.

**Reply by b20724964** [reply]
This feature still hasn't been added, right?

**Comment by snibgo** [comment]
Correct.
